operator/pkg/controllers/meshdeployment_controller_test.go

```diff
@@ -1,133 +0,0 @@
-/*
```

Comment:
Why did you delete the test case? I think it can help us verified the spec, and the result of reconciling is also correct.